MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+| Can manage release process of the project                                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can represent the project in public as a Maintainer                                                                                         |            ✗            |            ✗           |            ✓            |
+
+These roles are merit based. Refer to the corresponding section for specific
+requirements and the nomination process.
+
+## Contributor
+
+A Contributor invests time and resources to improve oneTBB project.
+Anyone can become a Contributor by bringing value in one of the following ways:
+  * Answer questions from community members.
+  * Propose changes to the design.
+  * Provide feedback on design proposals.
+  * Review and/or test pull requests.
+  * Test releases and report bugs.
+  * Contribute code, including bug fixes, features implementations,
+and performance optimizations.
+
+## Code Owner
+
+A Code Owner has responsibility for a specific project component or a functional
+area. Code Owners are collectively responsible, with other Code Owners,
+for developing and maintaining their component or functional areas, including
+reviewing all changes to corresponding areas of responsibility and indicating
+whether those changes are ready to be merged. Code Owners have a track record of
+contribution and review in the project.
+
+Requirements:
+  * Track record of accepted code contributions to a specific project component.
+  * Track record of contributions to the code review process.
+  * Demonstrated in-depth knowledge of the architecture of a specific project
+    component.
+  * Commits to being responsible for that specific area.
+
+The process of becoming a Code Owner is:
+1. A Contributor is nominated by opening a PR modifying the MAINTAINERS.md file
+including name, Github username, and affiliation.
+2. At least two specific component Maintainers approve the PR.
+3. CODEOWNERS file is updated to represent corresponding areas of responsibility.
+
+## Maintainer
+Maintainers are the most established contributors who are responsible for the 
+project technical direction and participate in making decisions about the
+strategy and priorities of the project.
+
+Requirements:
+  * Experience as a Code Owner.
+  * Track record of major project contributions to a specific project component.
+  * Demonstrated deep knowledge of a specific project component.
+  * Demonstrated broad knowledge of the project across multiple areas.
+  * Commits to using privileges responsibly for the good of the project.
+  * Is able to exercise judgment for the good of the project, independent of
+    their employer, friends, or team.
```

Comment:
how it can be measured though?

Reply:
Perhaps, maintainer would need to present proofs. 
I'm not sure how to measure this :)